GitUI/CommandsDialogs/FormStash.Designer.cs

```diff
@@ -269,6 +273,21 @@ private void InitializeComponent()
             this.Stashed.TabIndex = 2;
             this.Stashed.SelectedIndexChanged += new System.EventHandler(this.StashedSelectedIndexChanged);
             // 
+            // contextMenuStripStashedFiles
+            // 
+            this.contextMenuStripStashedFiles.Items.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.cherryPickFilesChangesToolStripMenuItem});
+            this.contextMenuStripStashedFiles.Name = "contextMenuStripStashedFiles";
+            this.contextMenuStripStashedFiles.Size = new System.Drawing.Size(209, 76);
+            // 
+            // cherryPickFilesChangesToolStripMenuItem
+            // 
+            this.cherryPickFilesChangesToolStripMenuItem.Image = global::GitUI.Properties.Resources.CherryPick;
+            this.cherryPickFilesChangesToolStripMenuItem.Name = "cherryPickFilesChangesToolStripMenuItem";
+            this.cherryPickFilesChangesToolStripMenuItem.Size = new System.Drawing.Size(208, 22);
+            this.cherryPickFilesChangesToolStripMenuItem.Text = "Cherry pick file\'s changes";
```

Comment:
Just a nitpick, but I'd prefer **not** including the _'s_ in _file's_. It seems to me that the possessive marker is generally used for "live" objects (like _dog's_, _girl's_, _John's_ etc.). However, I am not a native speaker so I might be wrong.

Reply:
>Just a nitpick, but I'd prefer not including the 's in file's.

I used the same text as the one already used elsewhere.
Is everyone ok to fix **all** the strings be part of this PR?